pkg/intcode/intcode.go

```diff
@@ -151,21 +117,21 @@ func (*multiply) run(p *program, arg1, arg2, arg3 int) bool {
 	return true
 }
 
-func (r *input) run(p *program, arg1, arg2, arg3 int) bool {
+func (*input) run(p *program, arg1, arg2, arg3 int) bool {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [parameter 'arg2' seems to be unused, consider removing or renaming it as _](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)